packages/firestore/test/integration/api/database.test.ts

```diff
@@ -426,6 +427,14 @@ apiDescribe('Database', persistence => {
       });
     });
 
+    it('inequality and array-contains on different fields works', () => {
+      return withTestCollection(persistence, {}, async coll => {
+        expect(() =>
+          coll.where('x', '>=', 32).where('y', arrayContainsOp, 'cat')
```

Comment:
I think this would be more readable if you hardcoded 'array-contains' here. It's part of the public API and should not change anytime soon.

Reply:
Agreed! This exists because 'array-contains' is not exposed in the .d.ts. So arrayContainsOp contains 'array-contains' casted to `WhereFilterOp`.  Once we expose the API, it can go away. I've cleaned up the TODO to make that clearer.